lib/Controller/WopiController.php

```diff
@@ -381,6 +397,111 @@ public function getFile(string $fileId, string $access_token): JSONResponse|Stre
 		}
 	}
 
+	#[NoAdminRequired]
+	#[NoCSRFRequired]
+	#[PublicPage]
+	#[FrontpageRoute(verb: 'GET', url: 'wopi/settings')]
+	public function getSettings(string $type, string $access_token): JSONResponse {
```

Comment:
Not a blocker: While this is close to the WOPI protocol, we may want to separate it in its own controller at some point.